docker/datahub-ingestion/Dockerfile

```diff
@@ -21,19 +21,20 @@ ARG PIP_MIRROR_URL
 RUN if [ "${PIP_MIRROR_URL}" != "https://pypi.python.org/simple" ] ; then pip config set global.index-url ${PIP_MIRROR_URL} ; fi
 ENV UV_INDEX_URL=${PIP_MIRROR_URL}
 
-COPY --chown=datahub ./metadata-ingestion /datahub-ingestion
-COPY --chown=datahub ./metadata-ingestion-modules/airflow-plugin /datahub-ingestion/airflow-plugin
+COPY --chown=datahub ./metadata-ingestion /metadata-ingestion
+COPY --chown=datahub ./metadata-ingestion-modules/airflow-plugin /metadata-ingestion/airflow-plugin
 
 ARG RELEASE_VERSION
-WORKDIR /datahub-ingestion
+WORKDIR /metadata-ingestion
 RUN sed -i.bak "s/__version__ = \"1\!0.0.0.dev0\"/__version__ = \"$(echo $RELEASE_VERSION|sed s/-/+/)\"/" src/datahub/__init__.py && \
     sed -i.bak "s/__version__ = \"1\!0.0.0.dev0\"/__version__ = \"$(echo $RELEASE_VERSION|sed s/-/+/)\"/" airflow-plugin/src/datahub_airflow_plugin/__init__.py && \
     cat src/datahub/__init__.py | grep __version__ && \
     cat airflow-plugin/src/datahub_airflow_plugin/__init__.py | grep __version__
 
 FROM base AS slim-install
 
-RUN uv pip install --no-cache -e ".[base,datahub-rest,datahub-kafka,snowflake,bigquery,redshift,mysql,postgres,hive,clickhouse,glue,dbt,looker,lookml,tableau,powerbi,superset,datahub-business-glossary]"
+RUN --mount=type=cache,target=/datahub-ingestion/.cache/uv,uid=1000,gid=1000 \
```

Comment:
Curious as to why this one is still `/datahub-ingestion` while it seems to be changed everywhere else?

Reply:
Maybe this is separating the `metadata-ingestion` source from the user's home directory

Reply:
Yup that's it
- `/datahub-ingestion` is the user's home directory - there's nothing in here other than dotfiles and the uv cache
- `/metadata-ingestion` has the code from the metadata-ingestion directory

I can make it restore the workdir